plugins/elixir

```diff
@@ -1 +1 @@
-repository = https://github.com/asdf-vm/asdf-elixir.git
+repository = https://github.com/glossia/mise-elixir.git
```

Comment:
can you move this into the mise-plugins org instead? that way it won't show a warning on install

Reply:
Definitively